spark/src/main/java/org/apache/iceberg/spark/source/Reader.java

```diff
@@ -145,6 +154,14 @@
     this.io = io;
     this.encryptionManager = encryptionManager;
     this.caseSensitive = caseSensitive;
+
+    boolean enableBatchReadsConfig =
+        options.get("iceberg.read.parquet-vectorization.enabled").map(Boolean::parseBoolean).orElse(true);
+    if (!enableBatchReadsConfig) {
+      enableBatchRead = Boolean.FALSE;
+    }
+    Optional<String> numRecordsPerBatchOpt = options.get("iceberg.read.parquet-vectorization.batch-size");
```

Comment:
Options passed into the read here should be short options because they come from the DataFrameReader interface:

```scala
spark.read.format("iceberg").config("snapshot-id", snapId).load("db.table");
```

The long option names like the ones you have here are what we use for table properties, which are tracked in `TableProperties` and documented. These options should default to the table property value, but a DataFrameReader option should override.

For short names, how about `vectorization-enabled` and `batch-size`? We should also add constants in `TableProperties` for the properties you have here. Let's turn off vectorization by default.

Reply:
I included this change in the PR against your branch.